packages/dapi/test/unit/grpcServer/handlers/tx-filter-stream/subscribeToTransactionsWithProofsHandlerFactory.spec.js

```diff
@@ -62,7 +50,14 @@ describe('subscribeToTransactionsWithProofsHandlerFactory', () => {
   let coreAPIMock;
   let getMemPoolTransactionsMock;
 
-  beforeEach(function beforeEach() {
+  // i do not know why, buy AcknowledgingWritable comes here as a sinon stub already
```

Comment:
I think it happens because you do `const writableStub = sinon.stub(AcknowledgingWritable.prototype, 'write');` and don't release it in `afterEach` hook. 
If you'd use `this.sinon`, it would've been released automatically as it is specified in `bootstrap.js`